corehq/apps/registration/views.py

```diff
@@ -478,9 +478,6 @@ def confirm_domain(request, guid=''):
         track_confirmed_account_on_hubspot.delay(requesting_user)
         request.session['CONFIRM'] = True
 
-        if settings.IS_SAAS_ENVIRONMENT:
-            # For AppCues v3, land new user in Web Apps
-            view_name = get_cloudcare_urlname(requested_domain.name)
```

Comment:
super nitpick: there's an unused import now

Reply:
less of a nitpick: would it be a good time to remove this code further up in this function?

https://github.com/dimagi/commcare-hq/blob/c0c208f18ecad9a97c267a7c1afc6a0a45ddc1e1/corehq/apps/registration/views.py#L433-L437